CP-CPS.md

```diff
@@ -228,9 +228,7 @@ No stipulation.
 
 ### 3.2.2 Authentication of organization identity
 
-ISRG only issues Domain Validation (DV) certificates. All FQDNs which will be listed in the Common Name and list of SANs in the certificate are fully validated prior to issuance.
-
-ISRG uses three methods for validating domain control:
+Prior to issuance of a Subscriber Certificate, ISRG uses the following methods to validate the Applicant's control of each FQDN listed in the Certificate:
```

Comment:
Maybe it would be good to say "uses at least one of the following methods" to make it clear that we don't perform all three. In the previous text we didn't have to say that because we were describing what we as an organization do, but now we're describing what happens in the issuance flow ("Prior to issuance...").

Reply:
Good call, that makes the phrasing much more natural.